### PR TITLE
nextbox: add documentation for soft reset

### PR DESCRIPTION
**Pull request by blu-nitro**
Todo:

- Release the soft reset changes in Nitrokey/nextbox#93

### CODE REVIEW

File: nextbox/technical/soft-reset.rst [hunk 1]
```diff
@@ -0,0 +1,28 @@
+Soft Reset
+=======================
+
+**If ever your NextBox doesn't start or behaves strangely the soft reset might fix your problems**
+The soft reset functionality tries to fix some common faliures on your NextBox.
+So whenever somethign is wrong, ie. the NextBox doesn't start, the first thing you could try is
```
Comment (daringer) [inline nextbox/technical/soft-reset.rst:6]: typo "somethign"
nitpick: "ie." is written as "i.e." here

File: nextbox/technical/soft-reset.rst [hunk 2]
```diff
@@ -0,0 +1,28 @@
+Soft Reset
+=======================
+
+**If ever your NextBox doesn't start or behaves strangely the soft reset might fix your problems**
+The soft reset functionality tries to fix some common faliures on your NextBox.
```
Comment (daringer) [inline nextbox/technical/soft-reset.rst:5]: typo: "faliures"

File: nextbox/faq/generic.rst [hunk 3]
```diff
@@ -1,6 +1,17 @@
 Generic FAQ
 ===========
 
+**Q:** What to do if my NextBox doesn't start?
+  If your NextBox doesn't start correctly or you have some other problem using
+  it, chances are a soft reset can fix it.
+  :doc:`../technical/soft-reset`. To press the button use a thin (~1mm) pin
+  like a smartphone sim-card tray removal device.
+  If this doesn't help, please make sure you have connected the NextBox correctly
```
Comment (daringer) [inline nextbox/faq/generic.rst:9]: first suggest a "factory-reset" then contact support ;)

**Review by daringer — CHANGES_REQUESTED** [review]

File: nextbox/technical/soft-reset.rst (hunk 1 above)
Comment (blu-nitro) [reply nextbox/technical/soft-reset.rst]: done

File: nextbox/technical/soft-reset.rst (hunk 2 above)
Comment (blu-nitro) [reply nextbox/technical/soft-reset.rst]: done

**Pull request merged**